channels-and-providers/sms/kannel.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: 'Kannel'
+description: 'Learn how to use the Kannel sms provider to send sms notifications using Novu'
+---
+
+Before integrating Kannel with Novu, you should have Kannel set up and configured as an SMS gateway on your server. Let’s look at how you can do that:
+
+# Setting up Kannel
+
+ 1. First, you need to install Kannel on your server. You can download it from the [official-website](https://www.kannel.org/download.shtml) or use a package manager specific to your operating system (e.g., apt-get for Ubuntu, yum for CentOS, etc.).
+ 2. Kannel's configuration is done through a file called kannel.conf. You need to edit this file to specify your SMS provider settings. Here's a basic configuration example:
+   
+   ```bash
+
+   group = smsc
+   smsc = smpp
+   smsc-id = YourSMSCID
+   host = SMSC_Hostname_or_IP
+   port = SMSC_Port
+   system-type = SMSC_System_Type
+   smsc-username = YourUsername
+   smsc-password = YourPassword
+   max-pending-submits = 10
+   allow-ip = "127.0.0.1"
+
+   ```
+   Replace the placeholders (e.g., **YourSMSCID**, **SMSC_Hostname_or_IP**, **SMSC_Port**, **SMSC_System_Type**, **YourUsername**, and **YourPassword**) with the actual values provided by your SMS provider.
+
+ 3. You can define services that will handle incoming and outgoing SMS messages. These services specify how Kannel should process SMS requests. Here's an example of an SMS service configuration:
+   
+   ```bash 
+
+   group = sendsms-user
+   username = YourUsername
+   password = YourPassword
+   concatenation = true
+   max-messages = 3
+
+   ```
+   Adjust the settings to your needs.
+
+ 4. Start Kannel with the following command:
+    
+    ```bash
+
+    bearerbox /path/to/kannel.conf
+    
+    ```
+    Make sure to replace /path/to/kannel.conf with the actual path to your Kannel configuration file.
+
+# Creating a Kannel integration with Novu
+
+  After setting up your Kannel follow these steps to integrate with novu:
+
+ 1. Visit the [Integrations Store](https://web.novu.co/integrations) on Novu.
+ 2. Locate **Kannel** and click on the `Disabled` button and mark it as `Active`.
```

Comment:
You omitted the Add a provider option. Please check a recent guide https://github.com/novuhq/docs/blob/staging/channels-and-providers/email/maqsam.mdx#L17 . The step 2 and 3 needs to be similar.